.github/dependabot.yml

```diff
@@ -6,3 +6,4 @@ updates:
       interval: "weekly"
     labels:
       - "CI"
+    target-branch: "3.8.x"
```

Comment:
It's unclear to me if Dependabot will start monitoring 3.8.x rightaway or after we merge this up into the default branch. Once it is merged up into the default branch, I think it will take other modifications to the file on that branch rightaway, so I think this is still the right move.